### PR TITLE
[netcore] Add some missing Sse/Sse2 intrinsics.

### PR DESCRIPTION
**Pull request by vargaz**
Add a set of generic OP_XOP_ opcodes to represent them instead of adding
a new opcode for each, the rest of the JIT doesn't care about the
exact operation.

### CODE REVIEW

**Comment by vargaz** [comment]
Will extend it later with more info.

File: src/mono/mono/mini/llvm-intrinsics.h [hunk 1]
```diff
@@ -0,0 +1,158 @@
+
+/*
+ * List of LLVM intrinsics
+ *
+ * INTRINS(name, llvm_name)
+ */
+
+INTRINS(MEMSET, "llvm.memset.p0i8.i32")
+INTRINS(MEMCPY, "llvm.memcpy.p0i8.p0i8.i32")
```
Comment (lpereira) [inline src/mono/mono/mini/llvm-intrinsics.h]: Any reason to use the 32-bit intrinsic for memcpy and the 64-bit version for memmove?

Comment (EgorBo) [reply src/mono/mono/mini/llvm-intrinsics.h]: @lpereira it's used to clear locals afair, we don't need i64 length there. `memmove` is used for C#'s `Buffer.Memmove` where it accepts IntPtr as a length.

Comment (lpereira) [reply src/mono/mono/mini/llvm-intrinsics.h]: Ah, makes sense.

**Comment by vargaz** [comment]
Will rebase on top of:
https://github.com/mono/mono/pull/19133

**Comment by vargaz** [comment]
Closing in favor of:
https://github.com/dotnet/runtime/pull/33356

**Pull request closed without merging**